File: C/senha_fixa.c
#include <stdio.h>
#include <locale.h>

int senha, tentativa;

int main() {
    setlocale(LC_ALL, "Portuguese");

    senha = 2002;

    printf("Digite a senha: ");
    scanf("%d", &tentativa);

    while (tentativa != senha) {
        printf("Senha inválida! Tente novamente: ");
        scanf("%d", &tentativa);
    }

    printf("Acesso permitido!");

}

File: Java/senha_fixa.java
import java.util.Locale;
import java.util.Scanner;

public class senha_fixa {

    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);
        Locale.setDefault(Locale.US);

        int senha = 2002;

        System.out.print("Digite a senha: ");
        int tentativa = sc.nextInt();

        while (tentativa != senha) {
            System.out.print("Senha Inválida! Tente novamente: ");
            tentativa = sc.nextInt();
        }

        System.out.print("Acceso permitido!");

        sc.close();

    }
}
